lib/fog/vsphere/models/compute/snapshot.rb

```diff
@@ -0,0 +1,42 @@
+require 'fog/compute/models/server'
+
+module Fog
+  module Compute
+    class Vsphere
+      class Snapshot < Fog::Model
+
+        identity  :ref
+        attribute :server_id
+
+        attribute :name
+        attribute :quiescedi, :default => false
+        attribute :description, :default => ''
+        attribute :create_time
+        attribute :power_state, :default => 'none'
+        attribute :ref
+        attribute :mo_ref
+        attribute :tree_node
+        attribute :snapshot_name_chain
+        attribute :ref_chain
+
+        attr_reader :service
+
+        def initialize(attributes = {})
+          super attributes
+        end
+
+        def child_snapshots(filters = {})
+          requires :mo_ref
+          service.snapshots(
+            { :server_id => server_id, :parent_snapshot => self }.update(filters)
+          )
+        end
+
+        def get(snapshot_ref)
```

Comment:
I agree, this method does the same as collection object (Snapshots#get). Therefore I prefer `get` signature. Even so, It could return self as well.

Shall I keep this signature? ...or do you prefer `find` or anything else instead?